leetcode/generate_md.py

```diff
@@ -1,3 +1,15 @@
-print('hello world!')
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 first step 
 Mistakes: 
 Dude...it is not even in right format.